crates/common/mqtt_channel/src/session.rs

```diff
@@ -40,6 +40,8 @@ pub async fn init_session(config: &Config) -> Result<(), MqttError> {
                 if Connection::pause_on_error(&err) {
                     Connection::do_pause().await;
                 }
+                eprintln!("Connection Error {}", err.to_string());
+                break;
```

Comment:
Since one breaks the loop there is no need to make a pause.

```suggestion
                eprintln!("Connection Error {}", err.to_string());
                break;
```

Reply:
ok, fixed.